modules/server/main.tf

```diff
@@ -569,11 +571,12 @@ resource "google_sql_database" "forseti-db" {
   instance = google_sql_database_instance.master.name
 }
 
-resource "google_sql_user" "root" {
-  name     = "root"
+resource "google_sql_user" "forseti_db_user" {
```

Comment:
@kevensen Depending on when this gets merged, you may need to update your PR to move this to the cloudsql module.